OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -1423,6 +1499,41 @@ function Install-OctopusDeploy {
         }
     }
 
+    if (![string]::IsNullOrEmpty($SSLCertificateThumbprint) -and ![string]::IsNullOrEmpty($SSLCertificateStoreName))
```

Comment:
This is in the wrong spot - we should be doing this up above before we restart the service.
We dont want to stop/start the service multiple times.
This is also pretty identical to the other code - )if it remains identical) it should move to a function